src/utils/hooks/api/vaults/use-get-vaults.tsx

```diff
@@ -39,6 +39,10 @@ const useGetVaults = ({ address }: { address: string }): VaultResponse => {
     if (!vaults || vaults.length === 0) return;
 
     for (const vault of vaults) {
+      if (vault.data === null) {
```

Comment:
What's the effect of this if one vault has data but the other doesn't?

Reply:
@tomjeatt sorry, I didnt mean to have this in my PR